Nasal/c172p.nas

```diff
@@ -642,6 +642,22 @@ setlistener("/sim/signals/fdm-initialized", func {
         }
     }, 1, 0);
 
+    # season-winter is a conversion value, see c172p-ground-effects.xml
+    if (getprop("/sim/startup/season") == "summer") {
+      setprop("/sim/startup/season-winter", 0);
+    }
+    else {
+        setprop("/sim/startup/season-winter", 1);
+    }
+    setlistener("/sim/startup/season", func (node) {
+        if (node.getValue() == "winter") {
```

Comment:
I would write code like this as follows:

```js
setprop("/sim/startup/season-winter", node.getValue() == "winter");
```

That's 1 line of code instead of 6.

Reply:
Thank you @onox

Reply:
Does this work as well then for the initial setting?
```js
setprop("/sim/startup/season-winter", getprop("/sim/startup/season") == "winter");
```